### ISSUE TITLE
Support Openstack Swift object store

### ISSUE THREAD

**Issue by airbots** [qubole/streamx#42]
Openstack Swift is also widely used object store. Many vendors like rackspace, Oracle, etc. It will be nice to support Openstack Swift as well.

**Comment by PraveenSeluka** [comment]
Hi @airbots Do you have your own deployment of Swift ? if hadoop file system for swift is stable, it will be easy to make it work with StreamX - do you have any experience with that ?

**Comment by airbots** [comment]
Yes, I create cloudstream (similar changes as streamx did for s3 based on hdfs connector) for kafka to access swift object store. It is in my github repository.
